assets/javascripts/initializers/add-canned-replies-ui-builder.js.es6

```diff
@@ -34,11 +34,18 @@ export default {
   initialize(container) {
     const siteSettings = container.lookup("site-settings:main");
     const currentUser = container.lookup("current-user:main");
-
+    let currentUserGroupNames;
+    if (currentUser && currentUser.groups) {
+      currentUserGroupNames = currentUser.groups.map(group => group.name);
+    }
+    const cannedRepliesGroups = Discourse.SiteSettings.canned_replies_groups;
```

Comment:
```suggestion
    const cannedRepliesGroups = siteSettings.canned_replies_groups.split("|").filter(x => x);
```

* We should be avoiding `Discourse` global whenever possible.
* List elements are joined with `"|"`, you have to split it back on the client-side.